scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -110,6 +110,48 @@ namespace <%= namespace %> {
             });
         }
 
+        private void CheckoutWithTokenizedPayment(TokenizedPaymentInput tokenizedPaymentInput, Action<ShopifyError> callback) {
```

Comment:
We might need to consider this later since any queries that return a checkout needs to be set as the `CurrentCheckout` in cart. Adding the extra layer means we need to make `CurrentCheckout` public or move it somewhere else